### PR TITLE
Bank 1.0

### PR DESCRIPTION
**Pull request by lachlanjc**

### CODE REVIEW

File: app/assets/stylesheets/organizer_positions.scss [hunk 1]
```diff
@@ -0,0 +1,3 @@
+// Place all the styles related to the OrganizerPositions controller here.
+// They will automatically be included in application.css.
+// You can use Sass (SCSS) here: http://sass-lang.com/
```
Comment (lachlanjc) [inline app/assets/stylesheets/organizer_positions.scss]: Please don’t commit these auto-generated stylesheets. We don’t organize our CSS by models/controllers & would be using SCSS partials if we did

File: app/controllers/organizer_positions_controller.rb [hunk 2]
```diff
@@ -0,0 +1,11 @@
+class OrganizerPositionsController < ApplicationController
+
+	def delete
+		@organizer_position = OrganizerPosition.find_by_id(params[:organizer_position_id])
```
Comment (lachlanjc) [inline app/controllers/organizer_positions_controller.rb]: Indentation here is off, should be using two spaces

File: app/models/organizer_position.rb [hunk 3]
```diff
@@ -1,7 +1,18 @@
 class OrganizerPosition < ApplicationRecord
+  scope :active, -> { where(deleted_at: nil) }
   belongs_to :user
   belongs_to :event
 
+  def delete!
+  	self.deleted_at = Time.current
```
Comment (lachlanjc) [inline app/models/organizer_position.rb]: Indentation (this is an issue on all the code you committed here)

File: config/routes.rb [hunk 4]
```diff
@@ -93,6 +93,10 @@
     resources :comments
   end
 
+  resources :organizer_positions, only: [] do
+      post 'delete', to: 'organizer_positions#delete'
```
Comment (lachlanjc) [inline config/routes.rb]: Shouldn’t be a `post` request for deleting data, should use `delete`

Comment (lachlanjc) [reply config/routes.rb]: On second thought, use `only: [:destroy]` on the `resources` line

Comment (tmb) [reply config/routes.rb]: I thought `destroy` was only for actually deleting something from the database. Is it applicable here?

File: app/models/organizer_position.rb (hunk 3 above)
Comment (tmb) [reply app/models/organizer_position.rb]: 👍 will fix

File: app/controllers/organizer_positions_controller.rb (hunk 2 above)
Comment (tmb) [reply app/controllers/organizer_positions_controller.rb]: 👍 will fix

File: app/assets/stylesheets/organizer_positions.scss (hunk 1 above)
Comment (tmb) [reply app/assets/stylesheets/organizer_positions.scss]: 👍

**Review by tmb — COMMENTED** [review]
we're going to need to run `User.find_each(&:save)` to generate slugs for current users when we deploy.

File: README.md [hunk 5]
```diff
@@ -1,3 +1,23 @@
...
+   ```
+4. Open [localhost:3000](http://localhost:3000)
+
+## Import database dump from Heroku
+
+```
+pg_restore --verbose --clean --no-acl --no-owner -h db -U postgres -d bank_development latest.dump
+```
+
+MIT License
```
Comment (lachlanjc) [inline README.md]: Bank is on MIT?

**Comment by tmb** [comment]
@lachlanjc: can you review the copy on my latest commit?

**Pull request merged**